### PR TITLE
Add documentation for oio-meta1-rebuilder and oio-meta2-rebuilder

### PR DESCRIPTION
**Pull request by acornard**

### CODE REVIEW

File: doc/source/admin-guide/operations_rebuild_meta.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+     | container      | container1                                                         |
+     | ctime          | 1530778169                                                         |
+     | max_versions   | Namespace default                                                  |
+     | objects        | 2                                                                  |
+     | quota          | Namespace default                                                  |
+     | status         | Enabled                                                            |
+     | storage_policy | Namespace default                                                  |
+     +----------------+--------------------------------------------------------------------+
+
+The format of the container IDs must be like ``43839DACDD060FA939FAE2714A60640BDC5AFFDDBE8C494BDAB7AA994C3190A5``
```
Comment (murlock) [inline doc/source/admin-guide/operations_rebuild_meta.rst]: Is it valid for `meta1` as it contains prefixes ?

File: doc/source/admin-guide/operations_rebuild_meta.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
+======================
+Rebuild a volume: meta
+======================
+
+This documentation is valid for meta1 and meta2. We use meta{1,2} in the following examples, but you need to replace by meta1 or meta2.
+
+Preparation
+~~~~~~~~~~~
+First you must run ``gridinit_cmd restart @meta{1,2}``, in this way you can check if meta services are up.
+You must wait to have a sufficient score for launch rebuilding.
```
[the remark this answers is not in the document]
Comment (jkasarherou) [reply doc/source/admin-guide/operations_rebuild_meta.rst]: `sufficient` ? high enough? how much?

File: doc/source/admin-guide/operations_rebuild_meta.rst [hunk 3]
```diff
@@ -0,0 +1,60 @@
+======================
+Rebuild a volume: meta
+======================
+
+This documentation is valid for meta1 and meta2. We use meta{1,2} in the following examples, but you need to replace by meta1 or meta2.
```
Comment (jkasarherou) [inline doc/source/admin-guide/operations_rebuild_meta.rst]: Might be clearer to separate this rebuild guide for meta1 and meta2, to discuss specifics about the two.

File: doc/source/admin-guide/operations_rebuild_meta.rst [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+You can check the score by running ``openio cluster list meta{1,2}``.
+
+Launch rebuilding
+~~~~~~~~~~~~~~~~~
+
+You can  launch the rebuild by using the ``oio-meta{1,2}-rebuilder`` tool:
+
+  .. code-block:: console
+
+    # oio-meta{1,2}-rebuilder <NAMESPACE>
```
Comment (jkasarherou) [inline doc/source/admin-guide/operations_rebuild_meta.rst]: Show at least one output example when running the rebuilder.

File: doc/source/admin-guide/operations_rebuild_meta.rst [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+     | container      | container1                                                         |
+     | ctime          | 1530778169                                                         |
+     | max_versions   | Namespace default                                                  |
+     | objects        | 2                                                                  |
+     | quota          | Namespace default                                                  |
+     | status         | Enabled                                                            |
+     | storage_policy | Namespace default                                                  |
+     +----------------+--------------------------------------------------------------------+
+
+The format of the container IDs must be like ``43839DACDD060FA939FAE2714A60640BDC5AFFDDBE8C494BDAB7AA994C3190A5`` for meta2.
```
Comment (jkasarherou) [inline doc/source/admin-guide/operations_rebuild_meta.rst]: Which format is it? You say it must be formatted like `base_name`, but the `.1` is missing at the end...

**Pull request merged**